client/client-lib/src/mint/mod.rs

```diff
@@ -994,6 +994,7 @@ mod tests {
                             let (_, nonce) = client
                                 .new_ecash_note(secp256k1_zkp::SECP256K1, amount, &mut dbtx)
                                 .await;
+                            dbtx.commit_tx().await.expect("DB Error");
```

Comment:
This change in particular, and the entire commit judging by the comment, seems to be a fixup, please squash.

Reply:
Will do.